src/test/scala/uk/gov/hmrc/perftests/disareturns/AuthRequests.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.perftests.disareturns
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import io.gatling.http.request.builder.HttpRequestBuilder
+import uk.gov.hmrc.performance.conf.ServicesConfiguration
+
+import scala.io.Source
+
+object AuthRequests extends ServicesConfiguration {
+
+  val authHost: String           = baseUrlFor("auth-login-stub")
+  val ggSignInUrl                = s"$authHost/application/session/login"
+  val authRequestPayload: String =
+    Source.fromResource("data/AuthRequestPayload.json").getLines().mkString
```

Comment:
Reading this from file looks like an overcomplication to me. Why isn't this just a defined model?